src/main/java/com/yahoo/bullet/parsing/PostAggregation.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright 2018, Oath Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.parsing;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import com.yahoo.bullet.common.BulletConfig;
+import com.yahoo.bullet.common.BulletError;
+import com.yahoo.bullet.common.Configurable;
+import com.yahoo.bullet.common.Initializable;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.Optional;
+
+import static com.yahoo.bullet.common.BulletError.makeError;
+import static java.util.Collections.singletonList;
+
+@Getter @Setter
+public abstract class PostAggregation implements Configurable, Initializable {
```

Comment:
Why does this need to be configurable?  I see configure() called in Query - but it does nothing in both PostAggregation classes (Computation and OrderBy).   Is this just in case we need to configure possibly more complicated post aggregations later?

Reply:
Yes. It's for future possible use.